addons/project/views/project_portal_project_task_templates.xml

```diff
@@ -115,8 +115,28 @@
             <t t-call="project.portal_tasks_list"/>
         </t>
     </template>
-
+    <template id="task_link_preview_portal_layout" inherit_id="web.frontend_layout" primary="True">
+        <xpath expr="//head/meta[last()]" position="after">
+            <t t-if="task">
+                <!-- Remove seo_object to not define og and twitter tags twice when wesbite is installed -->
+                <t t-set="seo_object" t-value="False"/>
```

Comment:
This seems risky if it's used for anything else in the original template, no? (I guess maybe it's not used for anything else right now)